### ISSUE TITLE
Rollbar integration

### ISSUE THREAD

**Issue by antgel** [RocketChat/Rocket.Chat#2317]
Hi, for us, it's really important. Is anybody working on this or planning to?

**Comment by engelgabriel** [comment]
Depends on #2426

**Comment by engelgabriel** [comment]
Please see initial documentation at https://github.com/RocketChat/Rocket.Chat/wiki/WebHook-Scripting---GitHub

**Comment by antgel** [comment]
Hey @engelgabriel, that's some pretty cool timing. FYI I'm also in touch with Rollbar support on this, they said it's not currently planned but they'll get it on the list, and that we could use webhooks in the meantime. :)

**Comment by engelgabriel** [comment]
Nice @antgel , tell them they can just use the save Slack integration they already have, just pointing to a different server.

**Comment by yanndupuy** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  dupuyyann@gmail.com
    Domain zyapps.com has exceeded the max emails per hour (451/400 (112%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext6.iad.github.net ([192.30.252.197]:53967 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by serveur.ex2-p10.com with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.86_1)
	(envelope-from <noreply@github.com>)
	id 1afmQV-000bi0-3G
	for yann.dupuy@zyapps.com; Tue, 15 Mar 2016 11:42:21 +0100
Date: Sun, 13 Mar 2016 09:35:09 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1457886909;
	bh=mB/3JV7nSr0vCP7+uFFDoIWgDfYzlovfN64uitn3cc4=;
	h=From:Reply-To:To:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=C7JZvAVR6FprwJ+OMDq5idRXYIrEZKhcOnDK6oQJS+eflDx2vLXZTMvDAihZ/obau
	 RRBVSF6jWb1ICYij0nyXyuUkIgVYBT1OPz6gtf045NKA65buGBZIvQJ32xJOphfDB3
	 Y/+gnppQq4c9zJUuk/FwdsxDtIKnkzhLkToDuI2Y=

**Comment by antgel** [comment]
@engelgabriel Hey, everybody's rocketchat server is a different URL, so I don't quite get what you mean by that. Can you explain? (rocketchat newbie here)

**Comment by engelgabriel** [comment]
When you go to Rollbar, it will ask you for the URL of the integration. Rather than giving the url generated on Slack admin panel, just give the url generated at your Rocket.Chat admin panel, they are compatible.

**Comment by antgel** [comment]
I tried that, it wasn't as you describe. I went to https://rollbar.com/<company>/<project>/notifications/slack, and it takes me to a screen like 
![image](https://cloud.githubusercontent.com/assets/295571/14351303/232e7500-fcd7-11e5-9441-459043afa13d.png) after which I click the Add to Slack button and it sends me to... Slack. (I don't have a Slack account.)

**Comment by engelgabriel** [comment]
They may have changed. Let me take a look.

**Comment by antgel** [comment]
@engelgabriel Any updates? Here's the latest from my thread with Rollbar:

> Thanks Antony,
> 
> I've made a note of your comment and will let you know if we end up working on this.  We're a small team and have a huge backlog, so even small investigations can take a while to bubble to the top of the list.
> 
> --
> ￼	Jesse from Rollbar
> On Wed, Aug 17, 2016 at 11:46 AM, "antgel" <antonyg@MUNGED.com> wrote: 
> Hi Jesse,
> 
> I hear you, but a couple of small points.
> 
> RocketChat is API-compatible with Slack, and it's open-source. 8484 stars on GitHub should give you an idea of where this could go.
> And I would imagine it's a relatively small change as all the Slack backend code is there - it's just a case of making the server name a variable instead of hard coded.
> 
> In the meantime, I have webhooks set up, but it's far from optimal (e.g. doesn't expose who performed an action).
> 
> On Tue, Aug 16, 2016 at 07:23 PM, "Jesse Gibbs" <jesse.gibbs@MUNGED.mail.intercom.io> wrote: 
> Hi Anthony,
> 
> This is still in our backlog but we're not planning to work on it yet.  It's not a very common request as most of our users prefer Hipchat or Slack.

It would be great if there was some way to move this forward.

**Comment by antgel** [comment]
@engelgabriel Any updates? :)

**Comment by TwizzyDizzy** [comment]
As discussed: please add a label "feature-needs-review".

/cc @SeanPackham 

Cheers
Thomas

**Comment by graywolf336** [comment]
I am going to close this one and have marked it as an App Idea. More information will be made public in the coming weeks about what exactly Rocket.Chat Apps are and how they can be used. :)
